packages/functional-tests/scripts/start-services.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -ex
+
+# This routine was formerly part of in test-ci.sh. It has been
+# split up so it can be run in separate steps in the CI,
+# resulting in more meaningful timing metrics.
+
+DIR=$(dirname "$0")
+
+cd "$DIR/../../../"
+
+mkdir -p ~/.pm2/logs
+mkdir -p artifacts/tests
+CI=true yarn workspaces foreach \
+    --verbose \
+    --topological-dev \
+    --include 123done \
+    --include browserid-verifier \
+    --include fxa-auth-server \
+    --include fxa-content-server \
+    --include fxa-graphql-api \
+    --include fxa-payments-server \
+    --include fxa-profile-server \
+    --include fxa-settings \
+    run start > ~/.pm2/logs/startup.log
```

Comment:
@ashrivastava-qa Just a call out here, because this is a new file and the diff might not be obvious. We no longer start fxa-shared or fxa-react. All the start operation does is run a build & watch. The build has already happened in the build job, so there is no reason to do this again.